### ISSUE TITLE
Yo Kraken:controller page  and test failed

### ISSUE THREAD

**Issue by Lioncode** [krakenjs/generator-kraken#161]
When creating a new controller it works great and routing is good. Unfortunately, when you run grunt test. It failed. This is what you get instead

```javascript
  1 passing (608ms)
  1 failing

  1) /page should say "hello":
     Error: expected body '<code><pre>{\n  "name": "page"\n}</pre></code>' to match /"name": "index"/
      at _stream_readable.js:944:16
```

I know this is easy to fix, by just replacing the code within test scripts or the controller output. But should this be also updated? Just saying when you creating a new controller , the test script should be working too?

**Comment by grawk** [comment]
I don't get that result. Can you please share your prompt options from the original app creation as well as the resultant app generation log?

**Comment by grawk** [comment]
OK. I got the result by generating an app like this:

```bash
Tell me a bit about your application:

? Description adf
? Author adf
? Template library? None
? Include i18n support? No
? Front end package manager ? No
? CSS preprocessor library? None
? JavaScript library? None
```

In this scenario the unit tests are looking at JSON responses, but not correctly looking for the name of the controller (name is "index" in all cases, which is only true for the default controller).

**Comment by grawk** [comment]
Fixed with `v2.2.1`
